sources/packages/web/src/views/institution/AppInstitution.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <div>
+    <router-view :key="$route.fullPath" />
```

Comment:
Please consider remove it. We can add once needed.

Reply:
AppStudent.vue has it so am keeping it for now(i understand we needed that because student profile had 2 modes). Can remove it in subsequent PR intended to fix the tests if it doesn't hurt anything else during testing